services/rest/websocket/ack.go

```diff
@@ -0,0 +1,38 @@
+package websocket
+
+import (
+	"time"
+
+	"github.com/odpf/raccoon/metrics"
+	"github.com/odpf/raccoon/serialization"
+	"github.com/odpf/raccoon/services/rest/websocket/connection"
+)
+
+var AckChan = make(chan AckInfo)
+
+type AckInfo struct {
```

Comment:
@AkbaraliShaikh If package itself is ack. This can be named just `Info` to remove brevity in usesage. `ack.AckInfo`

Reply:
right, the package name is sufficient to provide the context.

Reply:
I will create another PR to address this.

Reply:
The package name is not `ack` so the name is used as `AckInfo`. It's not a sub package.